common/src/main/java/com/wynntils/screens/base/widgets/WynntilsCheckbox.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.base.widgets;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.colors.CustomColor;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.Font;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.components.Checkbox;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+
+public class WynntilsCheckbox extends Checkbox {
+    private final int maxTextWidth;
+    private final int color;
+
+    public WynntilsCheckbox(
+            int x, int y, int width, int height, Component message, boolean selected, int maxTextWidth) {
+        super(x, y, width, height, message, selected);
+        this.maxTextWidth = maxTextWidth;
+        this.color = CommonColors.WHITE.asInt();
+    }
+
+    public WynntilsCheckbox(
+            int x,
+            int y,
+            int width,
+            int height,
+            Component message,
+            boolean selected,
+            int maxTextWidth,
+            CustomColor color) {
+        super(x, y, width, height, message, selected);
+        this.maxTextWidth = maxTextWidth;
+        this.color = color.asInt();
+    }
+
+    @Override
+    public void renderWidget(GuiGraphics guiGraphics, int mouseX, int mouseY, float partialTick) {
+        Minecraft minecraft = Minecraft.getInstance();
+        RenderSystem.enableDepthTest();
+        Font font = minecraft.font;
```

Comment:
```suggestion
        Font font = FontRenderer.getInstance().getFont();
```

(don't forget the import)